website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,13 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+          var cookieName = cookies[i].match(/([^=]+)(?=\=)/)[0];
+          document.cookie = cookieName + '= ; expires=Thu Jan 1 00:00:00 1970 GMT; path=/' + 
```

Comment:
I still don't see how this is correct. 

1) You're just setting `document.cookie` over and over again
2) You have a trailing `+`